src/lib/viewers/controls/color-picker/ColorPickerControl.scss

```diff
@@ -0,0 +1,44 @@
+@import '../styles';
+
+.bp-ColorPickerControl {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 48px;
+    height: 48px;
+}
+
+.bp-ColorPickerControl-button {
+    @include bp-ControlButton;
+}
+
+.bp-ColorPickerControl-palette {
+    $arrow-height: 12px;
+
+    position: absolute;
+    top: -#{$arrow-height};
+    left: 50%;
+    z-index: 1;
+    padding-bottom: 10px;
+    transform: translate(-50%, -100%);
+
+    &::after {
+        position: absolute;
+        left: 50%;
+        display: block;
+        border-top: $arrow-height solid $bp-controls-background;
+        border-right: $arrow-height solid transparent;
+        border-left: $arrow-height solid transparent;
+        transform: translateX(-50%);
+        content: '';
+    }
+}
+
+.bp-ColorPickerControl-swatch {
+    width: 16px;
```

Comment:
Should this be 18x18 or 20x20?

Reply:
I believe 16x16 is correct.

![Screen Shot 2020-12-03 at 1 30 25 PM](https://user-images.githubusercontent.com/7213887/101090585-bc308000-356b-11eb-9f4a-0e0f61543343.png)

Reply:
Yes, it should be 18x18, the border is taking up part of the space.